src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
```

Comment:
why did you reset paddings for body? looks like body doesn't have the default paddings
<img width="1335" alt="image" src="https://user-images.githubusercontent.com/30314756/206659143-a0eec50a-4a9e-4110-8fb6-170461e390aa.png">